source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+ANCOM is implemented in the ``q2-composition`` plugin. ANCOM assumes that few (less than about 25%) of the features are changing between groups. If you expect that more features are changing between your groups, you should not use ANCOM as it will be more error-prone (an increase in both Type I and Type II errors is possible). Because we expect a lot of features to change in abundance across body sites, in this tutorial we'll filter our full feature table to only contain gut samples. We'll then apply ANCOM to determine which, if any, sequence variants and genera are differentially abundant across the gut samples of our two subjects.
+
+ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. To build the composition artifact, a ``FeatureTable[Frequency]``  artifact must be provided to ``add-pseudocount`` (an imputation method), which will produce the ``FeatureTable[Composition]`` artifact.
+
+.. command-block::
+
+   qiime composition add-pseudocount \
+     --i-table table.qza \
+     --o-composition-table comp-table.qza
+
+We can then run ANCOM on the ``animations_subject`` column to determine what features differ in abundance across this metadata category.
+
+.. command-block::
+
+   qiime composition ancom \
+     --i-table comp-table.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-column animations_subject \
+     --o-visualization ancom-animations_subject.qzv
+
+.. question::
+   Which sequence variants differ in abundance across animations subject? In which subject is each sequence variant more abundant? What are the taxonomies of some of these sequence variants? (To answer the last question you'll need to refer to another visualization that was generated in this tutorial.)
+
+We're also often interested in performing a differential abundance test at a specific taxonomic level. To do this, we can collapse the features in our ``FeatureTable[Frequency]`` at the taxonomic level of interest, and then re-run the above steps. In this tutorial, we collapse our feature table at the genus level (i.e. level 6 of the Greengenes taxonomy).
+
+.. command-block::
+
+   qiime taxa collapse \
+     --i-table gut-table.qza \
+     --i-taxonomy taxonomy.qza \
+     --p-level 6 \
+     --o-collapsed-table gut-table-l6.qza
+
+   qiime composition add-pseudocount \
+     --i-table gut-table-l6.qza \
+     --o-composition-table comp-gut-table-l6.qza
+
+   qiime composition ancom \
+     --i-table comp-gut-table-l6.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-column Subject \
+     --o-visualization l6-ancom-Subject.qzv
+
+.. question::
+   Which genera differ in abundance across Subject? In which subject is each genus more abundant?
+
+
```

Comment:
as a conclusion, maybe link to the last section in the overview tutorial, which gives an overview of other plugins that can be used downstream

Reply:
k